### DISCUSSION TITLE
3.0.0 rc.1 证书路径变更在文档中没有注明, 从旧版升级上来的用户容易被误导

### DISCUSSION THREAD

**Discussion by MemoryShadow** [BeyondDimension/SteamTools#2908] [General]
在升级到该版本后, 我重新安装证书后浏览器一直报告证书不受信任.

在经过我的观察后发现, 该版本升级上来的用户证书路径中存在两个证书, 一个位于软件AppData的根目录中, 另一个位于软件AppData目录的`Plugins/Accelerator/`路径下. 该版本用户应当导入的是后者.

该变动没有在文档中得到体现.

相关信息:

- [#2843](https://github.com/BeyondDimension/SteamTools/issues/2843#issuecomment-1722230907)

**Comment by AigioL** [comment]
下个版本会删除旧目录的文件

https://github.com/BeyondDimension/SteamTools/blob/511a2fa36643730141e9d3123b58ea1709c297f8/src/BD.WTTS.Client/Migrations/Migrations.cs#L100-L140
